main.py

```diff
@@ -166,6 +176,14 @@ def _enrich_project_info_with_costs(projects):
     return projects
 
 
+def _enrich_project_info_with_orgs(client, projects):
+    for project in projects:
+        project['org']=_get_organization(client, project)
+        logger.debug('Organization root for Project %s: %s',project.get('projectId'), project.get('org'))
+    return projects
+    
+    
+    
```

Comment:
nit: 2 blank lines to separate functions.

Reply:
- removed extra line between functions 14a6234